components/user_list.tsx

```diff
@@ -1,37 +1,57 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, {ReactNode} from 'react';
 import {FormattedMessage} from 'react-intl';
 
 import Constants from 'utils/constants';
 import LoadingScreen from 'components/loading_screen';
 
-import UserListRow from './user_list_row';
+import {UserProfile} from '@mattermost/types/users';
 
```

Comment:
nit : no empty line inbetween

Reply:
If we remove this empty line then eslint shows an error.

Reply:
![image](https://user-images.githubusercontent.com/102028591/196387713-230eb4c7-44e6-4e64-83d6-251bc4e6301e.png)